docs/configuration/integrations/github-exporter-config.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "github_exporter_config"
++++
+
+# github_exporter_config
+
+The `github_exporter_config` block configures the `github_exporter` integration,
+which is an embedded version of
+[`github_exporter`](https://github.com/infinityworks/github-exporter). This allows for the collection of metrics from the github api.
+
+Full reference of options:
+
+```yaml
+  # Enables the github_exporter integration, allowing the Agent to automatically
+  # collect metrics for the specified github objects.
+  [enabled: <boolean> | default = false]
+
+  # Automatically collect metrics from this integration. If disabled,
+  # the github_exporter integration will be run but not scraped and thus not
+  # remote-written. Metrics for the integration will be exposed at
+  # /integrations/github_exporter/metrics and can be scraped by an external
+  # process.
+  [scrape_integration: <boolean> | default = <integrations_config.scrape_integrations>]
+
+  # How often should the metrics be collected? Defaults to
+  # prometheus.global.scrape_interval.
+  [scrape_interval: <duration> | default = <global_config.scrape_interval>]
+
+  # The timeout before considering the scrape a failure. Defaults to
+  # prometheus.global.scrape_timeout.
+  [scrape_timeout: <duration> | default = <global_config.scrape_timeout>]
+
+  # Allows for relabeling labels on the target.
+  relabel_configs:
+    [- <relabel_config> ... ]
+
+  # Relabel metrics coming from the integration, allowing to drop series
+  # from the integration that you don't care about.
+  metric_relabel_configs:
+    [ - <relabel_config> ... ]
+
+  # How frequent to truncate the WAL for this integration.
+  [wal_truncate_frequency: <duration> | default = "60m"]
+
+  # Monitor the exporter itself and include those metrics in the results.
+  [include_exporter_metrics: <bool> | default = false]
```

Comment:
We have copy pasted this everywhere but it's likely only valid for node and redis exporter, should be removed.

Reply:
Do you mean just the `include_exporter_metrics`?

Reply:
Do you mean just the `include_exporter_metrics` ?

Reply:
Yes